pgmax/fg/graph.py

```diff
@@ -249,7 +296,8 @@ def fg_state(self) -> FactorGraphState:
             factor_to_potentials_starts=copy.copy(self._factor_to_potentials_starts),
             factor_to_msgs_starts=copy.copy(self._factor_to_msgs_starts),
             log_potentials=self.log_potentials,
-            wiring=self.wiring,
+            enum_wiring=self.enum_wiring,
```

Comment:
I still think it's best to organize this inside some containers (e.g. a dictionary). This way, in the future if we add more types of factors we don't need to touch this line again (a sign of good abstraction/organization).